src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/EnhancedFurnaceListener.java

```diff
@@ -67,7 +69,10 @@ public void onItemSmelt(FurnaceSmeltEvent e) {
 
             if (state instanceof Furnace) {
                 FurnaceInventory inventory = ((Furnace) state).getInventory();
-                int amount = inventory.getSmelting().getType().toString().endsWith("_ORE") ? ((EnhancedFurnace) sfItem).getRandomOutputAmount() : 1;
+                Material smelting = inventory.getSmelting().getType();
+                boolean multiplier = SlimefunTag.ORES.isTagged(smelting) || SlimefunTag.RAW_METALS.isTagged(smelting);
```

Comment:
Looking at this, it may perhaps even be better to just create a new tag here which combines both of these.

Reply:
what should the new tag be called? `luck_multiplier_materials` or something else